x-pack/plugins/security_solution/common/api/entity_analytics/risk_engine/common.schema.yaml

```diff
@@ -33,13 +33,24 @@ components:
 
     Filter:
       description: An elasticsearch DSL filter object. Used to filter the risk inputs involved, which implicitly filters the risk scores themselves.
```

Comment:
I think we should find out where/how to raise issues with the type generator and raise one for these, the nwe can leave these commented out but link to the issue we create.

Reply:
The OpenAPI generator is maintained by @elastic/security-detection-rule-management team.
I created an issue and tagged them https://github.com/elastic/kibana/issues/181948

Reply:
Sorry to nit pick, we should create one for supporting  `oneOf` too, I believe this use is valid:

```
      # TODO Fix the following oneOf declaration
      # oneOf:
      #   - format: date
      #   - format: date-time
      #   - format: datemath
```

Reply:
I fixed it by using a different syntax:
https://github.com/elastic/kibana/pull/181715/files#diff-1142ed76ec48cb60fd68cc9c8d61add3f170f8b13701b173fb656502b4826984R45

Reply:
Ah nice 👍 @maximpn we have worked around an issue here, I am not sure if this was an error in the type generator or an invalid schema but the OpenAPI validator doesn't complain, shall we create a bug?